lib/rails_cursor_pagination/paginator.rb

```diff
@@ -99,10 +105,12 @@ def fetch(with_total: false)
     #   Optional, cannot be combined with `after`
     # @param order [Symbol]
     #   Optional, must be :asc or :desc
+    # @param record_decorator [Object, proc { |obj| obj.itself }]
+    #   Optional, must respond to `.call`
```

Comment:
Passing a `record_decorator` is optional in `#initialize`, but `#ensure_valid_params!` actually requires it.